devices/HapticGlove/include/SenseGloveHelper.hpp

```diff
@@ -0,0 +1,281 @@
+
+  /**
+   * Set the desired Force Feedback for all the fingers
+   * @param desiredValue desired force feedback of all the fingers
+   * @return true / false in case of success / failure
+   */
+  bool setFingersForceReference(const std::vector<double> &desiredValue);
+
+  /**
+   * Set the desired vibro-tactile feedback for all the fingers
+   * @param desiredValue desired vibro-tactile values
+   * @return true / false in case of success / failure
+   */
+  bool setBuzzMotorsReference(const std::vector<double> &desiredValue);
+
+  /**
+   * Set the desired vibro-tactile feedback for the palm
+   * @param desiredValue desired vibro-tactile value of the palm
+   * @return true / false in case of success / failure
+   */
+  bool setPalmFeedbackThumper(const int desiredValue);
+
+  /**
+   * Get the measured hand link poses values
+   * @param measuredValue measured joint values
+   * @return true / false in case of success / failure
+   */
+  bool getHandLinksPose(Eigen::MatrixXd &measuredValue);
+
+  /**
+   * Get the human hand joint angles
+   * @param jointAngleList std vector of doubles of human hand joint angles
+   * @return true / false in case of success / failure
+   */
+  bool getHandJointsAngles(std::vector<double> &jointAngleList);
+
+  /**
+   * Get the human hand joint angles
+   * @param measuredValue Eigen matrix of human hand joint angles
+   * @return true / false in case of success / failure
+   */
+  bool getHandJointsAngles(Eigen::MatrixXd measuredValue);
+
+  /**
+   * Get the glove link poses
+   * @param measuredValue Eigen matrix of glove poses
+   * @return true / false in case of success / failure
+   */
+  bool getGloveLinksPose(Eigen::MatrixXd &measuredValue);
+
+  /**
+   * Get the fingertip poses based on glove sensory data
+   * @param fingertipPoses Eigen matrix of glove poses [(pos: x, y, z) ,
+   * (rotation: w, x, y, z )]
+   * @return true / false in case of success / failure
+   */
+  bool
+  getGloveFingertipLinksPose(std::vector<std::vector<double>> &fingertipPoses);
+
+  /**
+   * Get the glove sensory data
+   * @param measuredValue vector of glove sensory data
+   * @return true / false in case of success / failure
+   */
+  bool getGloveSensorData(std::vector<float> &measuredValues);
+
+  /**
+   * Get the glove IMU data
+   * @param palmLinkPose human palm pose based on glove IMU data with the order
+   * pos(x y z), quat(w x y z)
+   * @return true / false in case of success / failure
+   */
+  bool getPalmLinkPose(std::vector<double> &palmLinkPose);
+
+  /**
+   * Trun off the vibro-tactile feedback
+   * @return true / false in case of success / failure
+   */
+  bool turnOffBuzzMotors();
+
+  /**
+   * Trun off the force feedback
+   * @return true / false in case of success / failure
+   */
+  bool turnOffForceFeedback();
+
+  /**
+   * get the number of buzz motors/vibro-tactile feedback motors
+   * @return number of buzz motors
+   */
+  int getNoOfBuzzMotors() const;
```

Comment:
using `Num` instead of `No` for the functions associated with getting number of elements ?